src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibraryWithOSMinimumVersion.cs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using FluentAssertions;
+using Xunit.Abstractions;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using System.Runtime.Versioning;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToBuildALibraryWithOSMinimumVersion : SdkTest
+    {
+        public GivenThatWeWantToBuildALibraryWithOSMinimumVersion(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Fact]
+        public void WhenPropertiesAreNotSetItShouldNotGenerateMinimumOSPlatformAttribute()
+        {
+            TestProject testProject = SetUpProject();
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            var runCommand = new DotnetCommand(Log, "run");
+            runCommand.WorkingDirectory = Path.Combine(testAsset.TestRoot, testProject.Name);
+            runCommand.Execute()
+                .Should()
+                .Pass().And.HaveStdOutContaining("NO ATTRIBUTE");
+        }
+
+        [Fact]
+        public void WhenPropertiesAreSetItCanGenerateMinimumOSPlatformAttribute()
+        {
+            TestProject testProject = SetUpProject();
+
+            var targetPlatformIdentifier = "iOS";
+            testProject.AdditionalProperties["TargetPlatformIdentifier"] = targetPlatformIdentifier;
+            testProject.AdditionalProperties["MinimumOSPlatform"] = "13.2";
```

Comment:
I'd suggest setting the TargetPlatformVersion to 14.0 for this test, instead of leaving it unset.